demo-bootstrap/src/test/java/io/microprofile/showcase/bootstrap/ParserTest.java

```diff
@@ -29,44 +29,47 @@
 public class ParserTest {
 
     @Test
-    public void testScheduleParser() {
-        URL resource = ParserTest.class.getClassLoader().getResource("schedule.json");
-        Assert.assertNotNull("Failed to load 'schedule.json'", resource);
+    public void testScheduleAndSpeakerParser() {
+        URL schedule = ParserTest.class.getClassLoader().getResource("schedule.json");
+        Assert.assertNotNull("Failed to load 'schedule.json'", schedule);
+
+        URL speaker = ParserTest.class.getClassLoader().getResource("speaker.json");
+        Assert.assertNotNull("Failed to load 'speaker.json'", speaker);
 
         Parser parser = new Parser();
-        BootstrapData data = parser.parse(resource);
+        BootstrapData data = parser.parse(schedule, speaker);
         Collection<Session> sessions = data.getSessions();
 
-        Assert.assertEquals(50, sessions.size());
+        Assert.assertEquals(100, sessions.size());
 
         Optional<Session> con5594 = sessions.stream()
-            .filter(s -> s.getCode().equals("CON5594"))
+            .filter(s -> s.getCode().equals("6E168E19-4B92-ECD8-01C6-6B4FC55C68FE"))
             .findAny();
 
-        Assert.assertTrue("Expected CON5594 in schedule", con5594.isPresent());
-        Assert.assertEquals("Cross-Functional Code Reviews", con5594.get().getTitle());
+        Assert.assertTrue("Expected specific session code in schedule", con5594.isPresent());
+        Assert.assertEquals("et tristique pellentesque, tellus", con5594.get().getTitle());
```

Comment:
Now that we are randomly generating data, this testcase is pretty specific to the content of the random data.  So, in the future, if anybody has to change this random data for any reason, then this test may no longer work.  I suppose that's okay, but it's kind of fragile.